apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -7,35 +7,37 @@ import {UnlocalizedTimeAgo as TimeAgo} from '@cdo/apps/templates/TimeAgo';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import firehoseClient from '@cdo/apps/lib/util/firehose';
 
-const measureElement = element => {
-  const DOMNode = ReactDOM.findDOMNode(element);
-  return {
-    height: DOMNode.offsetHeight
-  };
+const getElementHeight = element => {
+  return ReactDOM.findDOMNode(element).offsetHeight;
 };
 
-const initialCommentHeight = 40;
+const visibleCommentHeight = 30;
+
+const RubricPerformanceCopy = {
```

Comment:
(very minor) Not sure if there's a better word, but I was initially confused because I thought this object was a copy (i.e. duplicate) of something.

Reply:
Ah, good point. I'll change the name in my next PR